src/post/dto/create-post.dto.ts

```diff
@@ -0,0 +1,14 @@
+export class CreatePostDto {
+  postId: string;
+  title: string;
+  imageUrl: string;
+  description: string;
+  tag: [string];
```

Comment:
`tag: string[];`
for an array of strings.

Reply:
updated, changed name to "tags"